README.md

```diff
@@ -8,4 +8,23 @@ To build this, you need the following:
 - [`latexmk`](https://ctan.org/pkg/latexmk?lang=en) which may come with your LaTeX distribution
 - [Graphviz](https://graphviz.org/)
 
-After resolving these dependencies, run `make` to compile `spec.md` into `build/spec.pdf`.
+For compatability with continuous integration (CI) testing, use the versions of
+`pandoc` and `pandoc-crossref` that are [listed in the CI GitHub
+Action](.github/workflows/continuous-integration-ci.yml). If this release is
+*not* available in your package manager, you can download binaries from their
+GitHub releases pages:
+- [`pandoc` releases](https://github.com/jgm/pandoc/releases)
+- [`pandoc-crossref` releases](https://github.com/lierdakil/pandoc-crossref/releases)
+
+To run tests, you need Verilator and `firtool` available on your `$PATH`.
+
+After resolving these dependencies, use the following build targets:
+
+- `make` or `make all` will compile will compile `spec.md` and `abi.md` into
+  `build/spec.pdf` and `build/abi.pdf`.
+- `make format` will format all Markdown files by round-tripping them through
+  `pandoc`. *For this build step to be usable, use the exact versions of
+  `pandoc` and `pandoc-crossref` that CI uses!*
+- `make test` will extract FIRRTL and Verilog snippets from the specification
+  and ABI document and, respectively, run them through `firtool -parse-only` or
```

Comment:
And these versions need to match what's in CI as well (ofc).  I've never had a "problem" with either really (other than perhaps my firtool being too new for a short period but 🤷 and with your automation this is kept rather up-to-date in a timely manner) but anyway fine as-is but just thought I'd mention.

Reply:
Yeah, good point. I added some language in [196e997](https://github.com/chipsalliance/firrtl-spec/pull/320/commits/196e99782c800f4f47084eff7b55ba9bde238bc5) that indicates to go figure out what version of Verilator is being provided by the OSS CAD Suite. I do have failures (warnings) on latest Verilator which causes `make test` to not pass.